Sources/Ashton/AshtonHTMLWriter.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 import Foundation
-#if os(iOS)
+#if os(iOS) || (compiler(>=5.9) && os(visionOS))
```

Comment:
Writing it with the `compiler(>=5.9)` part prevents warnings about visionOS being an unknown platform when compiling using earlier versions of Swift.